services/QuillLMS/app/controllers/teachers/progress_reports_controller.rb

```diff
@@ -58,6 +54,16 @@ def diagnostic_insights
     render 'diagnostic_insights'
   end
 
+  def skills_report
+    @classrooms = classrooms
+
+    # TODO: update these two once we have more clearly scoped concepts to pull from
+    @all_skill_categories = Concept.where(parent_id: nil).order(:name)
+    @all_skill_sub_categories = Concept.where(parent_id: @all_skill_categories.ids).order(:name)
```

Comment:
We have a new helper pattern that we can use here, I believe: `Concept.level2` and `Concept.level1` should do this without having to specify ids directly.

Reply:
Yeah, this branch doesn't have those merged in yet. I will try to remember to change it, but we'll have to update it later when actually scope those concepts anyway.